### PR TITLE
Pass Debug option to a Blazor app for debugging

### LINKED ISSUE #13548: Shift+Alt+D Doesn't Do Anything
Shift+Alt+D Doesn't Do Anything. Additionally, no message is initially displayed in the console with the instructions as stated in the presentation by Steve Sanderson.

Chrome 75
Ubuntu 18.04.2
.NET Core 3.0.100-preview7-012821
Blazor 3.0.0-preview7.19365.7

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec2f61c7-9f51-6735-e328-3c7b495d70c3
* Version Independent ID: 4d06e565-282d-6387-aa90-cb2c2986d35a
* Content: [Debug ASP.NET Core Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/blazor/debug.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/debug.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #13548 

We can revert this instruction pending the outcome of the engineering issue (https://github.com/aspnet/AspNetCore/issues/12779). I've opened https://github.com/aspnet/AspNetCore.Docs/issues/13596 to track the reversion, and that will be monitored on the Blazor.Docs project.

### CODE REVIEW

File: aspnetcore/blazor/debug.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
 
 To debug a Blazor client-side app in Chrome:
 
-* Build a Blazor app in `Debug` configuration (the default for unpublished apps).
-* Run the Blazor app in Chrome (version 70 or later).
-* With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
-  * `Shift+Alt+D` on Windows/Linux
-  * `Shift+Cmd+D` on macOS
+1. Run a Blazor client-side app in `Debug` configuration. Pass the `--configuration Debug` option to the [dotnet run](/dotnet/core/tools/dotnet-run) command: `dotnet run --configuration Debug`.
+1. Access the app in Chrome (version 70 or later).
```
Comment (danroth27) [inline aspnetcore/blazor/debug.md]: It also now works with the Edge Dev Channel: https://www.microsoftedgeinsider.com

File: aspnetcore/blazor/debug.md [hunk 2]
```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
 
 To debug a Blazor client-side app in Chrome:
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/debug.md]: To debug a Blazor client-side app in Chrome or Edge:

File: aspnetcore/blazor/debug.md [hunk 3]
```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
...
 
-* Build a Blazor app in `Debug` configuration (the default for unpublished apps).
-* Run the Blazor app in Chrome (version 70 or later).
-* With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
-  * `Shift+Alt+D` on Windows/Linux
-  * `Shift+Cmd+D` on macOS
+1. Run a Blazor client-side app in `Debug` configuration. Pass the `--configuration Debug` option to the [dotnet run](/dotnet/core/tools/dotnet-run) command: `dotnet run --configuration Debug`.
+1. Access the app in Chrome (version 70 or later) or with a Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com)).
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/debug.md]: Remove that line. Why would you need it? How else are you going to access it under section?

Comment (guardrex) [reply aspnetcore/blazor/debug.md]: What? I don't follow. You have to use Chrome 70 or later or Edge preview.

File: aspnetcore/blazor/debug.md [hunk 4]
```diff
@@ -36,11 +36,11 @@ Development of further debugging scenarios is an on-going focus of the engineeri
...
+1. With the keyboard focus on the app (not in the developer tools panel, which you should probably close for a less confusing debugging experience), select the following Blazor-specific keyboard shortcut:
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/debug.md]: Way to long for MT. Break up in two sentences.  You should probably have this instruction earlier. Something like:
Debug with keyboard focus on the app, not the dev tools panel. The dev tools panel should typically be closed when debugging ...

File: aspnetcore/blazor/debug.md (hunk 3 above)
Comment (Rick-Anderson) [reply aspnetcore/blazor/debug.md]: Shouldn't that be a prereq then, listed earlier. I wouldn't be happy to find out only those versions work at this point.

Comment (guardrex) [reply aspnetcore/blazor/debug.md]: ok ... sounds good.

Comment (Rick-Anderson) [reply aspnetcore/blazor/debug.md]: To debug a Blazor client-side app in Chrome 70 or higher or Edge whaterver:

File: aspnetcore/blazor/debug.md (hunk 4 above)
Comment (Rick-Anderson) [reply aspnetcore/blazor/debug.md]: Or 
Client side debugging requires chrome 70 or later, or Edge Dev ...

**Pull request merged**